src/components/shift-schedule/ScheduleCellContent.tsx

```diff
@@ -0,0 +1,51 @@
+import { TableCell } from '@/components/ui/Table';
+import { cx } from '@/lib/utils';
+import { UserIcon, UsersIcon } from 'lucide-react';
+import { useTranslations } from 'next-intl';
+import type React from 'react';
+
+type ScheduleCellContentProps = {
+  members: {
+    name: string;
+  }[];
+};
+
+function ScheduleCellContent({ members }: ScheduleCellContentProps) {
+  const t = useTranslations('shiftSchedule.scheduleTable.scheduleCellContent');
+
+  const colorStyle =
```

Comment:
When setting styles always do it inline (That does sound unintuitive but it is actually easier to read). Also for all conditional styles that dont need the old style to still be set (only for some rare instances with transitions) you should use the cx function. This it because of some issues with tailwind when it comes to overwriting styles (I can explain this in detail later if needed)